admin/src/components/Help/HelpPanel.tsx

```
@@ -0,0 +1,167 @@
+								<span className="wppo-help-section__toggle">
+									<span
+										className={ `dashicons ${ activeSection === section.id ? 'dashicons-minus' : 'dashicons-plus-alt' }` }
+									></span>
+								</span>
+							</div>
+
+							{ activeSection === section.id && (
+								<div className="wppo-help-section__content">
+									<div
+										className="wppo-help-section__text"
+										dangerouslySetInnerHTML={ { __html: section.content } }
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)